inductiva/tasks/task.py

```diff
@@ -140,13 +142,35 @@ def kill(self) -> None:
         """
         self._api.kill_task(path_params=self._get_path_params())
 
+    def set_output_class(self, output_class):
+        """Set the output class of the task."""
+        self._output_class = output_class
+
+    def get_output(self, output_dir=None):
+        """Get the output of the task.
+
+        Returns:
+            The output of the task.
+        """
+
+        # Blocking call for the task to terminate
+        self.wait()
+        _, output_dir = api.download_output(self._api, self.id, output_dir)
+        output_dir = files.resolve_path(output_dir)
+
+        if self._output_class:
+            return self._output_class(output_dir)
+        else:
```

Comment:
No need for this else condition.

Reply:
Oh, no. I meant you just don't need the `else` bit.  You can simply have line 164 without it being in the `else` block.